www/docs/src/content/docs/0.1.0-beta.5/config-reference/overrides.md

```diff
@@ -0,0 +1,34 @@
+---
+title: overrides
+description: A reference page for overrides
+sidebar:
+  order: 6
+slug: 0.1.0-beta.5/config-reference/overrides
+---
+
+`overrides` is an object that is used to override the default configuration of the `astroStudioCMS`.
+
+## Usage
+
+```js title="astro.config.mjs"  {2-6}
+astroStudioCMS({
+  overrides: {
+    CustomImageOverride: "./src/components/MyCustomImage.astro",
+    FormattedDateOverride: "./src/components/MyCustomFormattedDate.astro",
+  },
+}),
+```
+
+### `CustomImageOverride`
+
+* **Type:** `string | undefined`
+* **Default:** `undefined`
+
+The path to a custom image component that will be used to render images in the `Astro-Studio-CMS`.
```

Comment:
Suddenly formatting it as `Astro-Studio-CMS` ???????????????????????????????????????

Reply:
This is the old version of the docs.... (beta.5)